documents/generic/service_test.go

```diff
@@ -0,0 +1,162 @@
+// +build unit
+
+package generic
+
+import (
+	"context"
+	"testing"
+
+	"github.com/centrifuge/go-centrifuge/anchors"
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/jobs"
+	"github.com/centrifuge/go-centrifuge/testingutils"
+	testinganchors "github.com/centrifuge/go-centrifuge/testingutils/anchors"
+	testingcommons "github.com/centrifuge/go-centrifuge/testingutils/commons"
+	testingconfig "github.com/centrifuge/go-centrifuge/testingutils/config"
+	"github.com/centrifuge/go-centrifuge/testingutils/testingjobs"
+	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/centrifuge/gocelery"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestService_CreateModel(t *testing.T) {
```

Comment:
IMO this test doesn't really test a generic doc, I would like to see it actually use custom attrs to store the fields.

Reply:
That is part of the testworld and documents test. This is just storing the generic doc

Reply:
TestCoreAPI_DocumentGenericCreateAndUpdate in the testworld and attributes does that since attributes are common to all the documents and the tests above are intended for specific document types alone